mgmt/rest/v2/metric.go

```diff
@@ -33,27 +33,53 @@ import (
 	"github.com/julienschmidt/httprouter"
 )
 
+// MetricsResp is the representation of metric operation response.
+//
+// swagger:response MetricsResponse
+type MetricsResp struct {
+	// in: body
+	Body struct {
+		Metrics []Metric `json:"metrics,omitempty"`
+	}
+}
+
+// MetricParams defines the input query parameters for get a specific metric.
+//
+// swagger:parameters getMetrics
+type MetricParams struct {
+	// in: query
+	Ns string `json:"ns"`
+	// in: query
+	Ver int `json:"ver"`
+}
+
 type MetricsResonse struct {
 	Metrics Metrics `json:"metrics,omitempty"`
 }
 
 type Metrics []Metric
 
+// Metric represents the metric type.
 type Metric struct {
-	LastAdvertisedTimestamp int64            `json:"last_advertised_timestamp,omitempty"`
-	Namespace               string           `json:"namespace,omitempty"`
-	Version                 int              `json:"version,omitempty"`
-	Dynamic                 bool             `json:"dynamic"`
-	DynamicElements         []DynamicElement `json:"dynamic_elements,omitempty"`
-	Description             string           `json:"description,omitempty"`
-	Unit                    string           `json:"unit,omitempty"`
-	Policy                  PolicyTableSlice `json:"policy,omitempty"`
-	Href                    string           `json:"href"`
+	LastAdvertisedTimestamp int64 `json:"last_advertised_timestamp,omitempty"`
+	//required: true
+	Namespace string `json:"namespace"`
```

Comment:
Do you agree the namespace of a metric should be required? Would you please let me know in what use case where a JSON/YAML representation won't specify the namespace for a metric?